lib/kredis/types/ordered_set.rb

```diff
@@ -17,6 +17,10 @@ def remove(*elements)
     zrem(types_to_strings(elements, typed))
   end
 
+  def include?(element)
+    zrank(type_to_string(element, typed)).present?
```

Comment:
I'd suggest using zscore instead which is O(1) instead of O(log(N)). I'd also avoid `.present?` in case an empty string is a valid set member.

```suggestion
    !!zscore(type_to_string(element, typed))
```

Reply:
Good points :+1: